src/intrusive-task-queue.js

```diff
@@ -185,4 +167,21 @@ class IntrusiveTaskQueue {
   }
 }
 
-export {IntrusiveTaskQueue};
+/**
+ * Returns an array containing the elements of this task queue in order. This
+ * is meant to be used for debugging and might be removed.
+ *
+ * @param {IntrusiveTaskQueue} queue
+ * @return {!Array<!Object>}
+ */
+function toArray(queue) {
+  let node = queue.head_;
+  const a = [];
+  while (node !== null) {
+    a.push(node);
+    node = node.tq_next_;
+  }
+  return a;
+}
+
+export {IntrusiveTaskQueue, toArray};
```

Comment:
Just a suggestion. It does seem handy for debugging, so maybe worth keeping around, but putting it on a separate export allows it to get tree shaken out of the distributed polyfill. OTOH, if you're debugging code using the distributed polyfill, this function won't be in there anymore.

Reply:
Good catch. TBH I can't remember the history here, and removing it would probably be fine (better?) since it's unused in code. But I like this.